scripts/kubestellar-snapshot.sh

```diff
@@ -420,9 +420,15 @@ for i in "${!cps[@]}" ; do # for all control planes in context ${context}
     cp_name[cp_n]=$name
     cp_ns[cp_n]="${cp_name[cp_n]}-system"
     cp_type[cp_n]=$(kubectl --context $helm_context get controlplane ${cp_name[cp_n]} -o jsonpath='{.spec.type}')
-    cp_pch[cp_n]=$(kubectl --context $helm_context get controlplane ${cp_name[cp_n]} -o jsonpath='{.spec.postCreateHook}')
-    cp_kubeconfig_content=$(get_kubeconfig "${helm_context}" "${cp_name[cp_n]}" "${cp_type[cp_n]}")
-    echo -e "- ${COLOR_INFO}${cp_name[cp_n]}${COLOR_NONE}: type=${COLOR_INFO}${cp_type[cp_n]}${COLOR_NONE}, pch=${COLOR_INFO}${cp_pch[cp_n]}${COLOR_NONE}, context=${COLOR_INFO}${cp_context[cp_n]}${COLOR_NONE}, namespace=${COLOR_INFO}${cp_name[cp_n]}-system${COLOR_NONE}"
+    cp_pchs=($(kubectl --context $helm_context get controlplane $name -o jsonpath='{.spec.postCreateHooks[*]}'))
+    cp_globalvars=$(kubectl --context $helm_context get controlplane $name -o jsonpath='{.spec.globalVars}')
+    # For backward compatibility, also try postCreateHook if postCreateHooks is empty
+    if [[ -z "${cp_pchs[*]}" ]]; then
+      cp_pchs=($(kubectl --context $helm_context get controlplane $name -o jsonpath='{.spec.postCreateHook}'))
+    fi
+    # Print control plane info
+    pch_list="${cp_pchs[*]}"
```

Comment:
Why introduce `pch_list`? Why not just use `cp_pchs` instead in the next command?